pkg/cmd/operator/gatherbase.go

```diff
@@ -166,3 +179,12 @@ func (o *GatherBaseOptions) GetPrinters() []collect.ResourcePrinterInterface {
 
 	return printers
 }
+
+// defaultExcludedSensitiveResourcesAsString returns a string representation of the default excluded sensitive resources for use in flag help.
+func defaultExcludedSensitiveResourcesAsString() string {
+	var ss []string
+	for _, gk := range collect.DefaultExcludedSensitiveResources() {
+		ss = append(ss, gk.String())
+	}
+	return strings.Join(ss, ", ")
+}
```

Comment:
What about just `fmt.Sprintf("%v", collect.DefaultExcludedSensitiveResources())` instead of iterating?

From my quick experiment it calls `String()` on the slice objects which should be enough I suppose?